source/code-snippets/usage-examples/insertMany.js

```diff
@@ -10,20 +10,20 @@ async function run() {
   try {
     await client.connect();
 
-    const database = client.db("sample_mflix");
-    const movies = database.collection("movies");
+    const database = client.db("insert_db");
```

Comment:
Question: did we decide the sample db should be called "insert_db"? Would we need different names for all the sample dbs? I wonder if it might be better to have a generic name like "myNewDb" or "myDb" (this placeholder choice has been around for at least a decade).
References:
https://docs.mongodb.com/manual/core/databases-and-collections/
https://docs.mongodb.com/v5.0/reference/method/Mongo/

Reply:
This was discussed in slack and captured in [this JIRA ticket](https://jira.mongodb.org/browse/DOCSP-18084)